cmd/ltagent/init.go

```diff
@@ -64,6 +65,43 @@ func createChannels(admin *userentity.UserEntity, numChannels int) error {
 	return nil
 }
 
+func createTeamAdmins(admin *userentity.UserEntity, numUsers int, config *loadtest.Config) error {
+	for i := 0; i < numUsers; i++ {
+		index := i * 10 // making every 10th user to be a team admin
```

Comment:
This is what I'd like to be configurable. The hard-coded 10 should be coming from `InstanceConfiguration`.